### PR TITLE
Update docs to show correct password-reset URL and fix formatting

### PR DESCRIPTION
**Pull request by hansemannn**
My PR is an: 💅 Enhancement

Main update on the: Documentation

This PR fixes the (required) URL used for password-resets via the REST-API. It also fixes some small typos and code-formatting.

### CODE REVIEW

**Comment by lauriejim** [comment]
Hello @hansemannn for this PR.

I don't know why but you create new lines for some sentences. There is a reason ?

**Comment by hansemannn** [comment]
@lauriejim I update the lines to match the markdown-recommendations of a limited number of chars per line to improve readability. If you don't want this, I can revert it.

File: docs/3.x.x/en/guides/authentication.md [hunk 1]
```diff
@@ -316,14 +320,13 @@ As for backend, we have a `switch...case` where we need to put our new provider
 Add the corresponding translation into: `packages/strapi-plugin-users-permissions/admin/src/translations/en.json`
 
 ```js
-  "PopUpForm.Providers.discord.providerConfig.redirectURL": "The redirect URL to add in your Discord application configurations",
+  "PopUpForm.Providers.discord.providerConfig.redirectURL": "The redirect URL to add in your Discord config",
```
Comment (lauriejim) [inline docs/3.x.x/en/guides/authentication.md]: Can you revert this update please.
If you check all translations files it's:

`"PopUpForm.Providers.discord.providerConfig.redirectURL": "The redirect URL to add in your Discord application configurations",`

`"PopUpForm.Providers.facebook.providerConfig.redirectURL": "The redirect URL to add in your Facebook application configurations",`

Comment (lauriejim) [reply docs/3.x.x/en/guides/authentication.md]: Then I will be ready to merge your PR :)

**Comment by hansemannn** [comment]
@lauriejim Done!

**Pull request merged**